test/test_jit.py

```diff
@@ -344,7 +344,6 @@ def __init__(self):
         self.bar = torch.jit.ScriptModule()
 
 
-@skipIfTorchDynamo()
```

Comment:
what is the point in running these with dynamo?

Reply:
We've caught some interesting errors with them. We could skip them entirely if we want to.